Maddori.Apple/Maddori.Apple/Global/UIComponent/KeywordLabel.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  KeywordLabel.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/18.
+//
+
+import UIKit
+
+enum KeywordType {
+    case defaultKeyword
+    case subKeyword
+    case disabledKeyword
+    case previewKeyword
+}
+
+final class KeywordLabel: UILabel {
+    
+    private let horizontalInset: CGFloat = 16.0
+    private let verticalInset: CGFloat = 13.0
+    
+    var keywordType: KeywordType?
+    
+    private var maskedCorners: CACornerMask {
+        switch keywordType {
+        case .previewKeyword:
+            return [.layerMaxXMaxYCorner, .layerMaxXMinYCorner, .layerMinXMinYCorner]
+        default:
+            return [.layerMinXMinYCorner, .layerMinXMaxYCorner, .layerMaxXMinYCorner, .layerMaxXMaxYCorner]
+        }
+    }
+    
+    var labelColor: [UIColor] {
+        switch keywordType {
+        case .defaultKeyword:
+            return [.gradientBlueTop, .gradientBlueBottom]
+        default:
+            return [.white100, .white100]
+        }
+    }
+    
+    private var customTextColor: UIColor {
+        switch keywordType {
+        case .defaultKeyword:
+            return .white100
+        case .disabledKeyword:
+            return .gray300
+        default:
+            return .gray600
+        }
+    }
+    
+    override var intrinsicContentSize: CGSize {
+        let size = super.intrinsicContentSize
+        return CGSize(width: size.width + 2 * horizontalInset, height: size.height + 2 * verticalInset)
```

Comment:
p3;

조금더 읽기 편하게 height 부분에서 한칸 줄 바꾸고 연산 순서대로 표시하는건 어떨까요 ??

```suggestion
        return CGSize(width: (horizontalInset * 2) + size.width,
                      height: (verticalInset * 2) + size.height)
```

Reply:
반영했습니다!
[d5e69be](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/18/commits/d5e69bec4e17b49194c625a3eb7991602461980c)